numpy/linalg/tests/test_linalg.py

```diff
@@ -423,25 +423,6 @@ def test_generalized_empty_herm_cases(self):
             exclude={'none'})
 
 
-def dot_generalized(a, b):
```

Comment:
Except not, as it turns out. This doesn't broadcast, but instead tries to work out which are vectors and which are matrices